static/js/components/CommentAttachmentPreview.jsx

```diff
@@ -153,6 +153,26 @@ const CommentAttachmentPreview = ({
   const isCached = commentId === cachedAttachmentCommentId;
 
   const [open, setOpen] = useState(false);
+
+  const getURLs = () => {
+    const type = filename.includes(".") ? filename.split(".").pop() : "";
+    let baseUrl = "";
+    // The FilePreviewer expects a url ending with .pdf for PDF files
+    if (associatedResourceType === "gcn_event") {
```

Comment:
This may be a good place for a [switch statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch).

Reply:
Agreed! Will add in a following PR. This was a quick fix I added here based on user feedback I got last night, but I want to have a dedicated PR. `react-file-preview` is a dead package, we should not use it anymore. And the PDF display is plain crap.

Reply:
Yes, there is a replacement now, linked from the related issue.

Reply:
Oh I didn't see that the last issue on that package was yours hahaha